.travis.yml

```diff
@@ -1,21 +1,8 @@
 language: node_js
+
 node_js:
-  - "6.0"
-  - "5.0"
-  - "4.0"
-  - "iojs"
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "0.8"
-  - "0.6"
+  - "7"
+  - "6"
+  - "4"
 
 sudo: false
-  
-before_install:
-  npm install -g npm@'>=1.4.3'
-
-matrix:
-  allow_failures:
-    - node_js: "0.8"
-    - node_js: "0.6"
```

Comment:
It was mostly dumb stuff like `require("babel-core").parse` no longer existing after the Greenkeeper update, and some unrelated changes to Babel 6 type definitions. As for `.travis.yml`, I just no longer care about Node pre-0.10 (those tests haven't been passing for a while, because of `^` version constraints in `package.json` files).